packages/kmod-6.1-nvidia/nvidia-tmpfiles.conf.in

```diff
@@ -1,2 +1,4 @@
 R __PREFIX__/lib/modules/__KERNEL_VERSION__/kernel/drivers/extra/video/nvidia/tesla - - - - -
 d __PREFIX__/lib/modules/__KERNEL_VERSION__/kernel/drivers/extra/video/nvidia/tesla 0755 root root - -
+C /usr/share/factory/etc/nvidia/fabricmanager.cfg - - - -
```

Comment:
This isn't ideal - systemd-tmpfiles copies files from `/usr/share/factory` into the path listed here. So this can just go back to being `/etc/nvidia/fabricmanager.cfg`, and the service unit can reference the config file in `/etc` instead of `/usr/share/factory/etc`.

Reply:
changed as suggested